quarkus/tests/integration/src/test/java/org/keycloak/it/storage/map/JPAStoreDistTest.java

```diff
@@ -33,7 +33,7 @@ public class JPAStoreDistTest {
     @Launch({ "start", "--optimized", "--http-enabled=true", "--hostname-strict=false" })
     void testSuccessful(LaunchResult result) {
         CLIResult cliResult = (CLIResult) result;
-        cliResult.assertMessage("Experimental feature enabled: map_storage");
+//        cliResult.assertMessage("Experimental feature enabled: map_storage");
```

Comment:
There's no longer CLI output at startup to warn for preview/experimental features, so not sure how to test this.